src/compas/artists/artist.py

```diff
@@ -89,6 +93,10 @@ def __new__(cls, *args, **kwargs):
         if not Artist.__ARTISTS_REGISTERED:
             register_artists()
             Artist.__ARTISTS_REGISTERED = True
+
+        if not args or len(args) < 1 or args[0] is None:
```

Comment:
don't think the length check is needed (already covered by `not args`).
we could also structure this differently:

1. `def __new__(cls, item, *args, **kwargs)` since `item` is mandatory
2. forget about `if not args`
3. add `if item is None` to raise ValueError

Reply:
yep, true: fixed here https://github.com/compas-dev/compas/pull/1009/commits/813c4674a138f31d8e87094cef1da9ca798747c5